src/util/charmrun-src/charmrun.C

```diff
@@ -4367,16 +4367,16 @@ static void start_nodes_daemon(std::vector<nodetab_process> & process_table)
   {
     const nodetab_host * h = p.host;
 
-    char *arg_currdir_r = pathfix(arg_currdir_a, h->pathfixes);
-    strcpy(task.cwd, arg_currdir_r);
-    free(arg_currdir_r);
-    char *arg_nodeprog_r = pathextfix(arg_nodeprog_a, h->pathfixes, h->ext);
-    strcpy(task.pgm, arg_nodeprog_r);
+    char *_arg_currdir_r = pathfix(arg_currdir_a, h->pathfixes);
```

Comment:
This should ideally be something more descriptive.

Reply:
Done.